release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,861 @@
+and can optionally provide a `ca` certificate bundle to use in requests to the
+server.
+
+See xref:../install_config/configuring_authentication.adoc#GitHub[Configuring
+Authentication and User Agent] for more information.
+
+[[ocp-311-sspi-connection-support-on-windows]]
+==== SSPI Connection Support on Microsoft Windows (Technology Preview)
+
+SSPI Connection Support on Microsoft Windows is currently in
+xref:ocp-311-technology-preview[Technology Preview] and not for production
+workloads.
+
+`oc` now supports the Security Support Provider Interface (SSPI) to allow for SSO
+flows on Windows. If you use the request header identity provider with a
+GSSAPI-enabled proxy to connect an Active Directory server to {product-title},
+users can automatically authenticate to {product-title} by using the `oc`  command
+line interface from a domain-joined Windows computer.
+
+See
+xref:../install_config/configuring_authentication.adoc#windows-sspi-using-request-header[Configuring
+Authentication and User Agent] for more information.
+
+[[ocp-311-notable-technical-changes]]
+== Notable Technical Changes
+
+{product-title} 3.11 introduces the following notable technical changes.
+
+[discrete]
+[[ocp-311-cluster-scoped]]
+==== subjectaccessreviews.authorization.openshift.io and resourceaccessreviews.authorization.openshift.io Are Cluster-scoped Only
+
+*_subjectaccessreviews.authorization.openshift.io_* and
+*_resourceaccessreviews.authorization.openshift.io_* are now cluster-scoped
+*only. If you need namespace-scoped requests, use
+**_localsubjectaccessreviews.authorization.openshift.io_* and
+*_localresourceaccessreviews.authorization.openshift.io_*.
+
+
+[discrete]
+[[ocp-311-scc-new-options]]
+==== New SCC options
+
+SCC has new options:
+
+
+* `AllowPrivilegeEscalation`
+* `DefaultAllowPrivilegeEscalation`
+* `forbiddenSysctls`
+* `allowedUnsafeSysctls`
+
+[discrete]
+[[ocp-311-oc-deploy-removed]]
+==== Removed oc deploy Command
+
+The `oc deploy` command, which was deprecated since {product-title} 3.7, is now
+fully removed. Use `oc rollout` instead.
+
+[discrete]
+[[ocp-311-pipeline-plugin-now-deprecated]]
+====  Pipeline Plug-in Is Deprecated
+
+The {product-title} Pipeline Plug-in is deprecated but continues to work with
+{product-title} versions up to version 3.11. For later versions of
+{product-title}, either use the `oc` binary directly from your Jenkins
+Pipelines, or use the {product-title} Client Plug-in.
+
+[discrete]
+[[ocp-311-logging-es5]]
+====  Logging: Elasticsearch 5
+
+Curator now works with Elasticsearch 5.
+
+See
+xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating
+Container Logs] for additional information.
+
+[discrete]
+[[ocp-311-oc-env-and-oc-volume-removed]]
+==== Removed oc env and oc volume Commands
+
+The deprecated `oc env` and `oc volume` commands are now removed. Use `oc set
+env` and `oc set volume` instead.
```

Comment:
For oc deploy, oc env, oc volume, need search all doc files if other places reference them and update them in a separate PR. This is to avoid they're still seen in other pages while 3.11 doc is announced.

Reply:
Thanks. This was done in https://github.com/openshift/openshift-docs/pull/11449